src/webviews/apps/plus/graph/GraphWrapper.tsx

```diff
@@ -270,6 +287,7 @@ export function GraphWrapper({
 							<GraphContainer
 								columnsSettings={graphColSettings}
 								cssVariables={styleProps.cssVariables}
+								getExternalIcon={getIconElementLibrary()}
```

Comment:
```suggestion
								getExternalIcon={getIconElementLibrary}
```

See https://github.com/gitkraken/vscode-gitlens/pull/2142#discussion_r943363065